app/assets/javascripts/components/ReportFilter.jsx

```diff
@@ -174,54 +208,50 @@ class ReportFilter extends React.Component {
-                      <div className="slider-title">
-                        NT { this.state.view_level } Z Score
+                    { (this.highest_nt_zscore && (this.lowest_nt_zscore !== this.highest_nt_zscore)) ? (
+                      <div className="">
+                        <div className="slider-title">
+                          NT Species Z Score
+                        </div>
+                        <div className="slider-values">
+                          <div suppressContentEditableWarning={true} contentEditable={true} className="nt_zscore start">
+                            { this.state.nt_zscore_start }
+                          </div>
+                          <div suppressContentEditableWarning={true} contentEditable={true} className="nt_zscore end">
+                            { this.state.nt_zscore_end }
+                          </div>
+                        </div>
+                        <div id="nt_zscore"></div>
                       </div>
-                      <div className="slider-values">
-                        <div className="nt_zscore start">{ this.state.nt_zscore_start }</div>
-                        <div className="nt_zscore end">{ this.state.nt_zscore_end }</div>
-                      </div>
-                      <div id="nt_zscore"></div>
-                    </div>
+                    ) : ''}
 
-                    <div className="">
-                      <div className="slider-title">
-                        NT { this.state.view_level } RPM
+                     { (this.highest_nt_rpm && (this.lowest_nt_rpm !== this.highest_nt_rpm)) ? (
+                      <div className="">
+                        <div className="slider-title">
+                          NT Species RPM
+                        </div>
+                        <div className="slider-values">
+                          <div onBlur={ this.updateThreshold } suppressContentEditableWarning={true} contentEditable={true} className="nt_rpm start">
+                            { this.state.nt_rpm_start }
+                          </div>
+                          <div  onBlur={ this.updateThreshold } suppressContentEditableWarning={true} contentEditable={true} className="nt_rpm end">
+                            { this.state.nt_rpm_end }
+                          </div>
+                        </div>
+                        <div id="nt_rpm"></div>
                       </div>
-                      <div className="slider-values">
-                        <div className="nt_rpm start">{ this.state.nt_rpm_start }</div>
-                        <div className="nt_rpm end">{ this.state.nt_rpm_end }</div>
-                      </div>
-                      <div id="nt_rpm"></div>
-                    </div>
+                    ) : ''}
+                     { (!this.has_valid_nt_rpm_range && !this.has_valid_nt_zscore_range) ?
+                    <div className="center"><small>Cannot set thresholds</small></div> : '' }
```

Comment:
Let's show this to Rebecca when it's pushed, she may prefer some different way to indicate the min=max condition.   Personally I don't think of that as an error condition.